Fixed dependencies

christianascone opened the pull request:
I fixed missing and broken dependencies.
Without them I was not able to run a fresh install.

nmoya commented on README.md:
```diff
@@ -0,0 +1,11 @@
+
+![](src/assets/img/hltb_icon.png)
+
+
```
Since you recently cloned the project, can you please list the steps to build the project?

christianascone replied:
Hi @nmoya. Yeah, I'm going to clone a fresh copy and re-build it in order to update README with every necessary step.

nmoya replied:
Thank you so much!

nmoya commented on app.json:
```diff
@@ -1,4 +1,4 @@
-  "displayName": "How Long To Beat"
-}
+  "name": "hltb",
+  "displayName": "hltb"
```
Why did we had to change this?

christianascone replied:
I think react changed this when I ran the project.
I will revert it because I suppose it does not need to be changed.

christianascone commented:
I fixed app name and displayName.
I also updated readme file with requirements and installation instructions (now it should runnable in a few steps).
